airflow/executors/kubernetes_executor.py

```diff
@@ -241,17 +253,23 @@ def _validate(self):
 
 class KubernetesJobWatcher(multiprocessing.Process, LoggingMixin):
     """Watches for Kubernetes jobs"""
-    def __init__(self, namespace, watcher_queue, resource_version, worker_uuid, kube_config):
+    def __init__(self,
+                 namespace: str,
+                 watcher_queue: 'Queue[KubernetesWatchType]',
+                 resource_version: str,
+                 worker_uuid: Optional[str],
+                 kube_config: Configuration):
         multiprocessing.Process.__init__(self)
-        self.namespace = namespace
-        self.worker_uuid = worker_uuid
-        self.watcher_queue = watcher_queue
-        self.resource_version = resource_version
-        self.kube_config = kube_config
+        self.namespace: str = namespace
```

Comment:
https://mypy.readthedocs.io/en/stable/class_basics.html#instance-and-class-attributes says we don't need to do this

Reply:
True, I was a bit over-protective here :)